project/src/test/java/com/google/sps/EventDetailTest.java

```diff
@@ -103,6 +107,34 @@ public void verifyGetAllEvents() throws IOException {
         CommonUtils.convertToJson(expectedEvents).trim(), stringWriter.toString().trim());
   }
 
+  @Test
+  public void verifyGetFilteredEvents() throws IOException {
+    Set<Event> expectedEvents = new HashSet(Arrays.asList(EVENT1, EVENT2));
+    Mockito.when(request.getParameter("labelParams")).thenReturn( "Work-Chess");
```

Comment:
I think you should create constant for "labelParams"

Reply:
done